### PR TITLE
Remove redundant checks in command modules

### PR DESCRIPTION
**Pull request by GeorgeCiesinski**
I removed checks for guild and guildId which was already being checked in `index.ts`. I also modified `postgres.ts` to stop outputting endless logs, but this can be disabled if needed.

### CODE REVIEW

File: .vscode/settings.json
Comment (Wissididom) [inline .vscode/settings.json:1]: I think the .vscode folder should be in the `.gitignore` file.

File: README.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ This bot was written to run locally or on docker with NodeJS and PostgreSQL and
         <a href="https://docs.docker.com/get-started/">Set up Docker</a> if you haven't already.
     </li>
     <li>
-        Run the bot with Docker Compose. You can do this by running either: <br><code>sudo docker-compose up -d</code><br> or if that doesn't work, then try: <br><code>sudo docker-compose --env-file docker.env up -d</code>
+        Run the bot with Docker Compose. You can do this by running either: <br><code>docker compose up -d</code><br>
```
Comment (Wissididom) [inline README.md:120]: I would rephrase that further because currently there is the word `either`, but just one command.

File: mod.ts [hunk 2]
```diff
@@ -9,22 +9,9 @@ import Database from "./data/postgres";
...
-            return;
-        }
-        if (!interaction.guildId) {
-            await interaction.reply({
-                content: 'Internal Server Error (score -> score -> interaction.guildId = null)',
-                ephemeral: true
-            });
-            return;
-        }
+
```
Comment (Wissididom) [inline mod.ts:12]: Do you think this empty line should be kept?

**Comment by GeorgeCiesinski** [comment]
Discovered 1 bug and one feature which can result in bug through user error. I tested the develop branch and confirmed these bugs existed in that branch as well. 

#108 & #109

**Review by Wissididom — COMMENTED** [review]
Just a few little things I noticed

File: README.md (hunk 1 above)
Comment (GeorgeCiesinski) [reply README.md]: Thanks for pointing that out. Corrected

File: mod.ts (hunk 2 above)
Comment (GeorgeCiesinski) [reply mod.ts]: Removed the line

File: .vscode/settings.json
Comment (GeorgeCiesinski) [reply .vscode/settings.json:1]: This might be useful to leave. The VsCode settings file only contains dictionary words that VsCode flags as spelled incorrectly. It should allow others to use VsCode without it whining about spelling errors.

Comment (Wissididom) [reply .vscode/settings.json:1]: Ok, then I'll approve,

I haven't used VS Code in this project yet because it wasn't complex enough that I needed to and I just used the terminal editor micro.

**Review by Wissididom — APPROVED** [review]
